sot/opcode_translator/executor/variable_dispatch.py

```diff
@@ -1027,3 +1031,35 @@ def dispatch_sum(var: ContainerVariable | TensorVariable, start: VariableBase =
         tracker=DummyTracker([var]),
     ),
 )
+
+
+def constant_numpy_equal(left, right):
+    numpy_ans = (left.get_py_value() == right.get_py_value(),)
+    return NumpyVariable(
+        numpy_ans,
+        left.graph,
+        tracker=DummyTracker([left, right]),
+    )
+
+
+Dispatcher.register(
+    operator.eq,
+    ("ConstantVariable", "NumpyVariable"),
```

Comment:
```suggestion
    ("ConstantVariable | NumpyVariable", "NumpyVariable"),
```

是不是可以加一个 `NumPy == NumPy` 的情况？